include/iso_alloc.h

```diff
@@ -8,7 +8,8 @@
 #include <stdlib.h>
 
 #if !__aarch64__ && !__x86_64__
-#pragma message "IsoAlloc is untested and unsupported on 32 bit platforms"
+#pragma error "IsoAlloc is untested and unsupported on 32 bit platforms"
```

Comment:
The error will stop the compilation process, making the `assert(sizeof(size_t) >= 64)` moot. You should put it outside of the condition I think.

Reply:
Also it should be 8, not 64 🤦‍♂️